README.md

```diff
@@ -149,9 +149,9 @@ These methods are available on a Writer object:
   Connect to the nsqd specified.
 * `close()` <br/>
   Disconnect from nsqd.
-* `publish(topic, msgs, [callback])` <br/>
+* `publish(topic, msgs, [timeMs], [callback])` <br/>
```

Comment:
I debated whether or not this should be a publish function just for delayed messages. If it were a separate function, it would be obvious that you can only publish a single message at a time. I'm not sure why it's this way in NSQ, but I'm sure there's a good reason. I'm kind of leaning towards separating it out. Thoughts?